localization_mode: "MCL"

init_particles: 2000

alpha1: 0.02
alpha2: 0.02
alpha3: 0.05
alpha4: 0.05

kld_epsilon: 0.01
kld_z: 0.99
min_particles: 100
max_particles: 5000